site/docs/hive.md

```diff
@@ -17,113 +17,211 @@
 
 # Hive
 
-## Hive read support
-Iceberg supports the reading of Iceberg tables from [Hive](https://hive.apache.org) by using a [StorageHandler](https://cwiki.apache.org/confluence/display/Hive/StorageHandlers). Please note that only Hive 2.x versions are currently supported.
+Iceberg supports reading and writing Iceberg tables through [Hive](https://hive.apache.org) by using a [StorageHandler](https://cwiki.apache.org/confluence/display/Hive/StorageHandlers).
+Only Hive 2.x versions are currently supported. 
```

Comment:
I think is not true.
We support Hive 3.1.2 too.

Also we might want to mention that for writes only MR is supported. Tez is only supported for reads only

Reply:
+1 for mentioning that Tez is only supported for reads, not writes. See a recent [issue](https://github.com/apache/iceberg/issues/2541) on this

Reply:
I will add a compatibility matrix then